isDeskOccupied.py

```diff
@@ -0,0 +1,303 @@
+            with open(self.args.uuid+'.csv') as csvInput:
+                inputReader = csv.reader(csvInput)
+                for row in inputReader:
+                    row = [int(i) for i in row]#parse each row into ints
+                    self.areas.append(row)#add each row to areas
+        except FileNotFoundError:
+            pass
+
+    #Define and save a new area from CLA
+    def newArea(self):
+        #Parse the area to add from commandline "(x,y)(x,y)"
+        str = self.args.newArea.split(')(')
+        newCoords = str[0].split('(')[1].split(',')
+        newCoords+=(str[1].split(')')[0].split(','))
+
+        with open(self.args.uuid+'.csv', 'a') as csvOutput:#write new area to uuid.csv, opening in 'a' mode creates a new file if one doesn't exist already
+            outputWriter = csv.writer(csvOutput)
+            outputWriter.writerow(newCoords)
+
+    #Open a commandline interface to remove an area, and save updated area list to uuid.csv
+    def removeArea(self):
+        try:
+            #Print all area coords as a menu, take input from user
+            with open(self.args.uuid+'.csv') as csvInput:
+                inputReader = csv.reader(csvInput)
+                i = 1
+                areas = []
+                for row in inputReader:
+                    areas.append(row)#add row to local list
+                    print(str(i)+": ("+row[0]+","+row[1]+"),("+row[2]+","+row[3]+")")#Print area bounds and associated index
+                    i+=1
+
+            if len(areas) == 0:#Return if there are no areas to remove
+                print("There are no areas to remove")
+                return
+
+            print("Enter the index of the area to remove:")
+            removeMe = int(input())#Take user input
+            areas.pop(removeMe-1)#Remove specified index from list
+
+            with open(self.args.uuid+'.csv', 'w') as csvOutput:#write updated area list to uuid.csv
+                outputWriter = csv.writer(csvOutput)
+                for row in areas:
+                    outputWriter.writerow(row)
+
+        except FileNotFoundError:
+            print("There are no areas to remove")
+    
+    #Check areas for occupancy
+    def checkAreas(self):
+        ##Summary
+        #api/event/getPolicyAlertGroupsForDevice -> event UUID
+        #Event UUID -> api/event.getPolicyAlertDetials -> Bounding Box Coords
+        #Bounding Box Coords vs. user defined areas -> occupancy
+        #For each Area:
+            #For each Human Motion Event:
+                #if 4 bounds contained: Area Occupied
+                #if >=2 bounds contained: Area May Be Occupied
+                #Else: Area Not Occupied
+
+        ##Get recent event groups
+        payload = {
+            "deviceUuid": self.args.uuid,
+            "maxResults": 10 #Get 10 most recent policy alerts grouped by time
+        }        
+        currentEvents = self.session.post(self.url+"event/getPolicyAlertGroupsForDevice",json = payload)
```

Comment:
Probably a better approach here would be to use the endpoint `/camera/getFootageBoundingBoxes` since those will be populated regardless of whether there is a policy set up to alert on human movement